index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Panda</title>
+    <link rel="stylesheet" href="panda.css">
+</head>
+<body>
+    <div class="nav">
+        <div class="nav_content">
+            <a id="nav1" href="/">
+                <img src="images/nav1.png">
+            </a>
+            <a id="nav2" href="/login">
+                <img src="images/nav2.png">
+            </a>
+        </div>
+    </div>
+    <div class="middle">
+        <div class="first">
+            <div class="text">
+                <h1>일상의 모든 물건을<br>거래해보세요</h1>
+                <a href="/items" class="look">구경하러가기기</a>
+            </div>
+            <img class="first_img" src="images/first_2.png">
+        </div>
+        <div class="second">
+            <img class="img" src="images/second.png">
+        </div>
+        <div class="third">
+            <img class="img" src="images/third.png">
+        </div>
+        <div class="fourth">
+            <img class="img" src="images/fourth.png">
+        </div>
+        <div class="last">
+            <img class="img" src="images/last.png">
+        </div>
+        <div class="footer">
+            <h3>©codeit - 2024</h3>
+            <div class="ppfq">
+                <a href="/privacy">
+                    <span>Privacy Policy</span>
+                </a>
+                <a href="/faq">
+                    <span>FAQ</span>
+                </a>
+            </div>
+            <div class="sns">
+                <a href="https://www.facebook.com"target="_blank">
+                    <img src="images/ic_facebook.png">
+                </a>
+                <a href="https://www.instagram.com"target="_blank">
+                    <img src="images/ic_instagram.png">
+                </a>
+                <a href="https://twitter.com"target="_blank">
+                    <img src="images/ic_twitter.png">
+                </a>
+                <a href="https://www.youtube.com"target="_blank">
+                    <img src="images/ic_youtube.png">
+                </a>
+                
+            </div>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="sns">
                <li>
                  <a href="https://www.facebook.com"target="_blank">
                      <img src="images/ic_facebook.png">
                  </a>
                </li>
                <li>
                  <a href="https://www.instagram.com"target="_blank">
                      <img src="images/ic_instagram.png">
                  </a>
                </li>
                <li>
                  <a href="https://twitter.com"target="_blank">
                      <img src="images/ic_twitter.png">
                  </a>
                </li>
                <li>
                  <a href="https://www.youtube.com"target="_blank">
                      <img src="images/ic_youtube.png">
                  </a>
                </li>
            </ul>
```